gnd/src/main/res/menu/nav_drawer_menu.xml

```diff
@@ -21,6 +21,10 @@
         android:id="@+id/nav_join_project"
         android:icon="@drawable/ic_menu_project"
         android:title="@string/select_project_menu_item"/>
+    <item
+        android:id="@+id/nav_offline_maps"
```

Comment:
In hindsight, I actually like "Offline maps" for the class names as well (OfflineMapsSelector), since it'll be consistent with the UI and is even more self-explanatory. If you agree, feel free to ignore this comment and we can update it in a follow up PR.

Reply:
I'll open a separate PR after this one to do the refactor.